<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emergencyPayment</class>
 <widget class="QMainWindow" name="emergencyPayment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="outstandingAfterDiscountAmount">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>800</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Outstanding pay: </string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Points:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="points">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="redeemPointsButton">
       <property name="text">
        <string>Redeem points</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="paymentSuccessfulLabel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>390</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Payment successful.</string>
    </property>
   </widget>
   <widget class="QLabel" name="messageToUser">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>random text</string>
    </property>
   </widget>
   <widget class="QLabel" name="cardNumberError">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>220</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="outstandingPayLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>800</string>
    </property>
   </widget>
   <widget class="QLabel" name="CashPromptLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Please pay at the reception</string>
    </property>
   </widget>
   <widget class="QLabel" name="CVVError">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>270</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>*in this case, the outstanding pay is equivalent to
the total price for the operations you underwent in the ER.  </string>
    </property>
   </widget>
   <widget class="QLabel" name="redeemPointsMessage">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="paymentMethodCombo">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>E-Wallet</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Insurance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cash</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Credit</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="balanceToDisplay">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Payment</string>
    </property>
   </widget>
   <widget class="QLabel" name="BalanceLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Balance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="WalletDiscountLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>E_Wallet discount: </string>
    </property>
   </widget>
   <widget class="QLabel" name="outstandingAfterDiscountLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Outstanding pay after discount: </string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>203</width>
      <height>136</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="CardNumberLabel">
       <property name="text">
        <string>Card number</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="CVVTxbx"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="CardNameLabel">
       <property name="text">
        <string>Card name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="CardNumberTxbx"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="CardNameTxbx"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ExpiryDateLabel">
       <property name="text">
        <string>Expiry date</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="CVVlabel">
       <property name="text">
        <string>CVV</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="ExpDateEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Payment method</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PayButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>360</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pay</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Type of Payment:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="typeOfPaymentToDisplay_2">
       <property name="text">
        <string>Emergency visit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="discountAmount">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>800</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
